File: crates/find-dup-defs/tests/fixtures/mixed/py/core_b.py
MAX_RETRIES = 5
DEFAULT_TIMEOUT = 30


type UserIds = list[int]


def compute_score(values, weight):
    total = 0
    for v in values:
        total += v * weight
    return total / len(values)


def plus_numbers(x, y):
    result = x + y
    return result * 2


class Repository:
    def fetch_item(self, item_id):
        record = self.store.get(item_id)
        if record is None:
            raise KeyError(item_id)
        return record

    @staticmethod
    def normalize(name):
        cleaned = name.strip().lower()
        return cleaned.replace(" ", "_")

File: crates/find-dup-defs/tests/fixtures/mixed/ts/core_b.ts
export const MAX_RETRIES = 5;
export const DEFAULT_TIMEOUT = 30;

export type UserIds = number[];

export interface Repo {
    get(id: number): number;
    set(id: number, value: number): void;
    has(id: number): boolean;
}

export function computeScore(values: number[], weight: number): number {
    let total = 0;
    for (const v of values) {
        total += v * weight;
    }
    return total / values.length;
}

export function plusNumbers(x: number, y: number): number {
    const result = x + y;
    return result * 2;
}

export class Repository {
    fetchItem(itemId: number): number {
        const record = this.store.get(itemId);
        if (record === undefined) {
            throw new Error("missing");
        }
        return record;
    }
}
